src/Servers/Kestrel/test/FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -108,7 +108,6 @@ public static IEnumerable<object[]> LargeUploadData
             }
         }
         [Theory]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore-internal/issues/2489")]
         [MemberData(nameof(LargeUploadData))]
         public async Task LargeUpload(long? maxRequestBufferSize, bool connectionAdapter, bool expectPause)
```

Comment:
Same deal. Does https://github.com/dotnet/aspnetcore/issues/21638 have insufficient metadata? Where is the actual fix recorded? (It looks like @anurse closed it w/ no action.)

Reply:
There isn't an actual fix recorded. I'm just following the protocol. If you disagree with the protocol for un-quarantining tests, we should bring it up at DoI.

From what it looks like, LargeUpload has passed fairly consistently for both sockets and Libuv

Reply:
I'm not disagreeing with the docs. We made an executive decision to unquarantine some test failures for which no data exists but had met the 30 day requirement. Go ahead and merge once @sebastienros's typo fix clears validation.